ds3-sdk/src/main/java/com/spectralogic/ds3client/helpers/LoggingFileObjectPutter.java

```diff
@@ -0,0 +1,23 @@
+package com.spectralogic.ds3client.helpers;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.nio.channels.SeekableByteChannel;
+import java.nio.file.Path;
+
+public class LoggingFileObjectPutter implements Ds3ClientHelpers.ObjectChannelBuilder {
+    final private FileObjectPutter objectPutter;
+    final private static Logger LOG = LoggerFactory.getLogger(LoggingFileObjectPutter.class);
+
+    public LoggingFileObjectPutter(final Path inputDirectory) {
```

Comment:
If we really want this to be extensible we should change it so that the other object putter is passed in, instead of it being created inside of this class.  That will allow it be to used with other `FileObjectPutter` classes.